virttest/libvirt_vm.py

```diff
@@ -2938,6 +2938,15 @@ def prepare_guest_agent(self, prepare_xml=True, channel=True, start=True):
         self.install_package('pm-utils', ignore_status=True, timeout=15)
         self.install_package('qemu-guest-agent')
 
+        self.set_state_guest_agent(start)
+
+    def set_state_guest_agent(self, start):
```

Comment:
this change implicitly make functions such as  _is_ga_running() to be hidden. You may need check whether any python has ever used them.